### DISCUSSION TITLE
Deprecated method

### DISCUSSION THREAD

**Discussion by prashant-ERA** [airbnb/lottie-ios#2491] [General]
## Which Version of Lottie are you using?

Lottie 2.5.0

## Actual Behavior

I am using Lottie-ios in my iOS app and during code review it has been highlighted that the library uses deprecated method sprintf. I am using version 2.5.0 of the repository. Please advice if you plan to update the deprecated method to the latest method in the updated version of the repository.

**Comment by calda** [comment]
2.5.0 is very old. The latest version (4.5.0) doesn't use `sprintf`.
